src/topology/bounded_continuous_function.lean

```diff
@@ -429,5 +429,89 @@ def of_normed_group_discrete {α : Type u} {β : Type v}
   (f : α  → β) (C : ℝ) (H : ∀x, norm (f x) ≤ C) : α →ᵇ β :=
 of_normed_group f C H continuous_of_discrete_topology
 
+/- Formula for norm. -/
+lemma norm_eq (f : α →ᵇ β) :
+  ∥f∥ = Inf {C : ℝ | 0 ≤ C ∧ ∀ (x : α), ∥f x∥ ≤ C} :=
+begin
+  rw [← sub_zero f, ← dist_eq_norm, bounded_continuous_function.dist_eq],
+  simp,
+end
+
 end normed_group
+
+section normed_space
+/-! In this section, if `β` is a normed space, then we show that the space of bounded
+continuous functions from `α` to `β` inherits a normed space structure, by using
+pointwise operations and checking that they are compatible with the uniform distance. -/
+
+variables {𝕜 : Type*} [nondiscrete_normed_field 𝕜]
+variables [topological_space α] [normed_group β] [normed_space 𝕜 β]
+variables {f g : α →ᵇ β} {x : α} {C : ℝ}
+
+instance : has_scalar 𝕜 (α →ᵇ β) :=
+⟨λc, λf, ⟨λx, c • f.1 x,
+  continuous.smul continuous_const f.2.left,
+  begin
+    cases f.2.right with C hbound,
+    use ∥c∥ * C,
+    intros,
+    have hnneg : 0 ≤ ∥c∥ := norm_nonneg c,
+    specialize hbound x y,
+    rw dist_eq_norm at hbound ⊢,
+    calc ∥c • f x - c • f y∥ = ∥c • (f x - f y)∥ : by rw smul_sub c (f x) (f y)
+    ... = ∥c∥ * ∥f x - f y∥ : norm_smul c (f x - f y)
+    ... ≤ ∥c∥ * C : mul_le_mul_of_nonneg_left hbound hnneg
+  end⟩⟩
+
+instance : module 𝕜 (α →ᵇ β) :=
+  module.of_core $
```

Comment:
The style is to indent the *type* when it goes on a new line,
but *term*s are not indentend. So everything should move 2 spaces to the left.

Reply:
Thanks, fixed.